sw/device/tests/BUILD

```diff
@@ -3033,6 +3033,167 @@ opentitan_test(
     ],
 )
 
+_RV_DM_JTAG_LC_STATES = get_lc_items(
+    CONST.LCV.TEST_UNLOCKED0,
+    CONST.LCV.DEV,
+    CONST.LCV.RMA,
+)
+
+[
+    opentitan_test(
+        name = "rv_dm_jtag_{}".format(lc_state),
+        srcs = ["example_test_from_flash.c"],
+        cw310 = cw310_jtag_params(
+            otp = "//hw/ip/otp_ctrl/data:img_{}".format(lc_state),
+            tags = [
+                "fpga_cw310_sival",
```

Comment:
Hm... It looks like none of these tests are actually running in CI. "fpga_cw310_sival" is not one of the tags the jobs look for, and these tests have none of the ones they do:

- Test ROM tests: "cw310_test_rom"
- ROM tests: "cw310_rom_with_fake_keys", "cw310_rom_with_real_keys"
- Manufacturing tests: "manuf"

We probably need to figure out how we want these tests to work with CI and tag filtering. Should this get any tag other than "cw310_rom_with_fake_keys"?

Reply:
@cfrantz @moidx @timothytrippel ^ (for some possibly interested parties)

Reply:
I followed #20402. Should we update the CI to include the fpga_cw310_sival tag?